src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java

```diff
@@ -70,9 +71,16 @@ public static void setUp() {
         eventId = "eventId";
         calendar = new Calendar();
         calendar.setId("calendarId");
-        event = new Event();
+        EventEntity auxEventEntity = new EventEntity("id1", "Summary", "Description", "Spain", "2",
+            10, 5, "1996-01-08T10:20:30", "1996-01-08T12" +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['5' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)